tests/index/test_index_finder.cpp

```diff
@@ -72,7 +72,7 @@ SEQAN_TYPED_TEST_CASE(IndexFinderTest, UnidirectionalIndexTypes);
 SEQAN_TYPED_TEST(IndexFinderTest, FindFirstChar)
 {
     SEQAN_ASSERT(find(this->finder, prefix(concat(this->text), 1u)));
-    SEQAN_ASSERT(find(this->finder, (const char*)"A"));
+    SEQAN_ASSERT(find(this->finder, "A"));
```

Comment:
I didn't notice the asterix :grin:

@h-2 Should I simply allow ``TPattern`` to be a pointer in ``find()``? Because `(const char*)` didn't implement the ContainerConcept.